README.rst

```diff
@@ -1202,7 +1202,7 @@ Choose from the list of available rules:
 
   Properties should be set to ``null`` instead of using ``unset``.
 
-  *Risky rule: changing variables to ``null`` instead of unsetting them will mean they still show up when looping over class variables. With PHP 7.4, this rule might introduce ``null`` assignments to property whose type declaration does not allow it.*
+  *Risky rule: risky when relying on attributes to be removed using ``unset`` rather than be set to ``null``.Changing variables to ``null`` instead of unsetting means these still show up when looping over class variables.With PHP 7.4, this rule might introduce ``null`` assignments to properties whose type declaration does not allow it.*
```

Comment:
The issue is mainly when using reference property (property converted to a reference property by assigning it as a reference using `&`).

see https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4939#issuecomment-670524770

this PR should mention reference property explicitly

Reply:
I think you understand the issue far better than me. I've read your issue several times and this is the best I can do! 

If you can provide better copy changes than this I'm very happy to apply it there, but otherwise this will be it and I will merge the PR and close your issue. Docs are indeed important but if no one can provide better than suggestions than there is no point in leaving those open as an `issue`, thanks for understanding.

Reply:
https://3v4l.org/jkdck - see reference propertly with `= null` vs `unset()`

Reply:
thanks! as I stated, I know your issue, I've read your issue, I've looked at your new example. I think my PR is spot on!
If you disagree and have better _copy_ (so not example and links and suggestion) for the docs, please let me know, otherwise I merged as is, thanks!